### PR TITLE
ADD: dynamic sequence length mostly working

### PR DESCRIPTION
**Pull request by benleetownsend**

### CODE REVIEW

File: finetune/target_models/ordinal_regressor.py [hunk 1]
```diff
@@ -138,22 +138,21 @@ def _get_input_pipeline(self):
...
         featurizer_state["features"] = tf.abs(
             tf.reduce_sum(featurizer_state["features"], 1)
         )
-        return ordinal_regressor(
-            hidden=featurizer_state["features"],
-            targets=targets,
-            n_targets=n_outputs,
+        if 'context' in featurizer_state:
+            featurizer_state["context"] = tf.abs(
+                tf.reduce_sum(featurizer_state["context"], 1)
```
Comment (madisonmay) [inline finetune/target_models/ordinal_regressor.py]: Here's where we need to mask yea?

Comment (benleetownsend) [reply finetune/target_models/ordinal_regressor.py]: But in every target_model

Comment (madisonmay) [reply finetune/target_models/ordinal_regressor.py]: :+1:
